lefthook.yml

```diff
@@ -0,0 +1,14 @@
+# EXAMPLE USAGE:
+#
+#   Refer for explanation to following link:
+#   https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md
+#
+  pre-commit:
+    parallel: true
+    commands:
+      server:
+        root: server
+        run: npm run format
+#      client:
+#        root: client/src
+#        run: npm run format
```

Comment:
I deactivated the client directory format since it needs your review. If you want it to be active, remove this comment out after fixing the format.